### ISSUE TITLE
detect jspm app name configuration for html templates

### ISSUE THREAD

**Issue by olanod** [aurelia/framework#11]
Here is my jspm custom configuration in my package.json.
```
{
  "jspm": {
    "name": "my-app",
    "directories": {
      "baseURL": "src",
      "lib": "src/app",
      "packages": "vendor"
    },
    "dependencies": {
      "aurelia-bootstrapper": "^0.9.1"
    }
  }
}
```
Application scripts are in the app folder(e.g. app.js) and when I want to import them they are prefixed with 'my-app', like `import {App} from 'my-app/app';`. So I had to add my app module to the `aurelia-app` attribute: `aurelia-app='my-app/app'` and it worked, my app script is being imported, the problem is with the associated template, it tries to download it from `/my-app/app.html` but it's in `/app/app.html`.  
I fixed it for now implementing `getViewStrategy` and returning an absolute path to my view but I think this should be considered.

**Comment by Serhioromano** [comment]
I would run `jsmp init -p` and reconfigure everything. Basicaly it should be 

```json
"directories": {
      "baseURL": "my-app",
      "lib": "my-app/app",
      "packages": "vendor"
}
```

What I did from skeleton app, 

1. I created folder `build` and moved all site files there line `index.html`, `config.js`, ... 
2. I created folder `build/assets/jsmp` and configured it to be package folder. 
3. I changed `package.json` to point into that folder.
4. run `jsmp install` to move everything into that folder.
5. I changed `gulpfile.js` to build JS files into `build/modules`
6. I changed `gulpfile.js` ro run server from `build`

What I get is a clean build folder which contain only files used by site. All those `gulpfile.js` and `.gitignore` and even `src` folder with original JS files and other items are outside of this. So this folder is ready for deployment. Just `rsync` in into your server with `flightplan.js` or copy to `s3` static site. Almost no post production required.

![2015-01-29_10-23-20](https://cloud.githubusercontent.com/assets/650741/5951336/f27b9532-a7a0-11e4-8fb1-2ae4dd295562.png)
